src/MongoDB.Driver.Core/Core/NativeLibraryLoader/ILibraryLocator.cs

```diff
@@ -17,6 +17,7 @@ namespace MongoDB.Driver.Core.NativeLibraryLoader
 {
     internal interface ILibraryLocator
     {
+        bool IsX32ModeSupported { get; }
```

Comment:
This is a temporal property to support x32 mode with snappy. This property should be overridden in Snappy32NativeMethods.cs and Snappy64NativeMethods

Reply:
cc: @rstam

Reply:
no problem